template-processors/base/bin/processParameters.sh

```diff
@@ -51,8 +53,8 @@ if [ -n "${FOLDERS}" ]; then
     YAML_FILES="$(find "${DIR}" -maxdepth 1 -name \*.json -o -name \*.yaml -o -name \*.yml)"
 
     # merge the files
-    if [ ! -z "${YAML_FILES}" ]; then
-      goyq merge -i -x "${VALUES_FILE}" ${YAML_FILES}
+    if [ -z "${YAML_FILES}" ]; then
+      goyq merge -i -x "${VALUES_FILE}" "${YAML_FILES}"
```

Comment:
on this line, we cannot use `"` around YAML_FILES, as it is important that we pass all values from YAML_FILES as separate arguments to goyq; please use a different approach that preserves this functionality